src/graphics/TFTDisplay.cpp

```diff
@@ -484,16 +496,21 @@ bool TFTDisplay::connect()
 #endif
 
     tft.init();
+
 #if defined(M5STACK)
     tft.setRotation(0);
+#elif defined(RAK14014)
+    tft.setRotation(1);
+    tft.setSwapBytes(true);
+    tft.fillScreen(TFT_BLACK);
 #elif defined(T_DECK) || defined(PICOMPUTER_S3)
     tft.setRotation(1); // T-Deck has the TFT in landscape
 #elif defined(T_WATCH_S3)
     tft.setRotation(2); // T-Watch S3 left-handed orientation
 #else
     tft.setRotation(3); // Orient horizontal and wide underneath the silkscreen name label
-#endif
     tft.fillScreen(TFT_BLACK);
+#endif
     return true;
```

Comment:
That change eliminates the fillScreen call for all other variants which is a break. What happens if you put the call back after the endif and remove it in the RAK14014 condition?

Reply:
Thanks for the catch.   Fixed and recompiled/works on the 10701.